jobs/jobs.go

```diff
@@ -314,10 +314,10 @@ func (job *Job) cleanup(ctx context.Context, cancel context.CancelFunc) {
 		}
 	}
 	cancel()
-	job.Unsubscribe(job.Bus) // deregister from events
+	job.exec.CloseLogs()
```

Comment:
Double checking this method and I noticed the following...

```go
func (c *Command) CloseLogs() {
	if c != nil && c.logger != nil {
```

Does the `c != nil` need to be there?